src/builder/cleanup.js

```diff
@@ -0,0 +1,10 @@
+import { join } from "path"
+import { unlinkSync } from "fs"
+
+export default (wd, fileToKeep, assets) => {
+  assets
+    .filter((file) => file !== fileToKeep)
+    .forEach((file) => {
+      unlinkSync(join(wd, file))
```

Comment:
https://ci.appveyor.com/project/MoOx/phenomic/build/1580#L2452

Path on windows is different from linux. ('/' vs '\')